priv/repo/migrations/20170326201602_add-user-group-is-admin.exs

```diff
@@ -0,0 +1,21 @@
+defmodule :"Elixir.Pairmotron.Repo.Migrations.Add-user-group-is-admin" do
+  use Ecto.Migration
+
+  def up do
+    alter table(:users_groups) do
+      add :is_admin, :boolean
+    end
+
+    execute "UPDATE users_groups SET is_admin = FALSE where is_admin IS NULL"
+
+    alter table (:users_groups) do
+      modify :is_admin, :boolean, null: false, default: false
```

Comment:
nope, honestly for some reason I thought that this had to be done in two steps because I did a similar (but not the same) thing somewhere else. Derp.